common/module.hpp

```diff
@@ -79,7 +79,7 @@ class Module : public rclcpp::Node {
 								 const std::string& topic,
 								 const rclcpp::Logger& logger);
 								 
-	bool requestDataDictInitialization(int maxRetries = 3);
+	bool requestDataDictInitialization(int maxRetries = 5);
```

Comment:
Relates to the previous comment, has it been a problem with initialization with DataDictionary in the passed?